modular_skyrat/modules/xenoarch/code/modules/research/xenoarch/xenoarch_reward.dm

```diff
@@ -0,0 +1,76 @@
+GLOBAL_LIST_INIT(tier1_reward, list(
+	/obj/item/xenoarch/useless_relic = 5,
+	/obj/item/candle = 1,
+	/obj/item/trash/chips = 1,
+	/obj/item/trash/raisins = 1,
+	/obj/item/trash/cheesie = 1,
+	/obj/item/trash/candy = 1,
+))
+
+GLOBAL_LIST_INIT(tier2_reward, list(
+	/obj/item/xenoarch/broken_item/tech = 1,
+	/obj/item/xenoarch/broken_item/plant = 1,
+	/obj/item/xenoarch/useless_relic = 5,
+))
+
+GLOBAL_LIST_INIT(tier3_reward, list(
+	/obj/item/xenoarch/broken_item/weapon = 3,
+	/obj/item/xenoarch/broken_item/illegal = 1,
+	/obj/item/xenoarch/broken_item/alien = 1,
+	/obj/item/stack/spacecash/c10000 = 1,
+))
+
+
+
+GLOBAL_LIST_INIT(tech_reward, list(
+	/obj/item/pipe_dispenser = 1,
+	/obj/item/construction/rcd = 1,
+	/obj/item/anomaly_neutralizer = 1,
+	/obj/item/megaphone = 1,
+	/obj/item/bodybag/bluespace = 1,
+	/obj/item/relic = 1,
+	/obj/item/disk/tech_disk/spaceloot = 1,
+	/obj/item/raw_anomaly_core/random = 1,
+	/obj/item/bag_of_holding_inert = 1,
```

Comment:
Probably not a good idea to be able to drop both a raw core and inert BoH; one or the other is fine though.